documentation/modules/post/windows/gather/credentials/foxmail.md

```diff
@@ -0,0 +1,53 @@
+## Verification Steps
+
+  1. Download the latest installer of Foxmail.
+  2. Use foxmail to login email server.
+  3. Remember to save the account password.
+  4. Get a `meterpreter` session on a Windows host.
+  5. Do: ```run post/windows/gather/credentials/foxmail```
+  6. If the account file is saved in the system, the email, server, port and plaintext password will be printed.
+
+## Options
+
+ **ACCOUNT_PATH**
+
+  - Specifies the Account directory path for Foxmail
+
+## Scenarios
+
```

Comment:
@cn-kali-team You didn't fill in the version number of Windows in your update. Can you please update this? I'd like to see something in the format of `### FoxMail 7.2 on Windows 10 v1903`. Otherwise if an issue crops up on a specific version of Windows, it will be harder to track down what the expected output should be.

Reply:
Thanks for updating this @cn-kali-team! 👍